features/support/env.rb

```diff
@@ -42,6 +42,15 @@
   log.error(message + "failed")  if scenario.failed?
 end
 
+After('@users') do
+  user_name = config["features"]["users"]["user_name"]
+  users = control_node.openstack.user.list
+  user_names = users.map(&:name)
+  if user_names.include? user_name
+    control_node.openstack.user.delete user_name
+  end
```

Comment:
This might be written shorter by using `find`:
```ruby
if control_node.openstack.user.list.find {|u| u.name == user_name }
  control_node.openstack.user.delete(user_name)
end
```